packages/pwa-kit-react-sdk/src/ssr/universal/contexts/index.js

```diff
@@ -1,9 +1,12 @@
 /*
- * Copyright (c) 2021, salesforce.com, inc.
+ * Copyright (c) 2022, Salesforce, Inc.
  * All rights reserved.
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
 import React from 'react'
-export default React.createContext()
+
+export const DeviceContext = React.createContext()
+export const PageRequestContext = React.createContext()
+export const PageResponseContext = React.createContext()
```

Comment:
Thoughts on the naming of the contexts and hooks. Calling them "page" request and "page" response to me implies that they are or might be other types of requests/responses. 

Naming is hard.. but do you think that useServerRequest and useServerResponse is a little better? The name almost implies that you probably aren't going to get a value using it on the client.. and having server in the name is a little more connected to what the res/req objects are. 

The following comment isn't completely important, but could we possibly combine the 2 into something called useServerContext.. it might give us more room to grow in the future without wrapping components around components. I'm not married to that tho.

Reply:
I like the name `useServerContext`. Like you said, it's clear that these are coming from the server. And the name is more future-proof, allowing for additional properties if need be.